src/app/components/form/form.component.ts

```diff
@@ -41,7 +41,7 @@ export class FormComponent implements OnInit {
             keywords: sleutelwoorden
         };
 
-        this.http.post(environment.api + '/searchrequest', body).subscribe((data: any) => alert(data.texts));
+        this.http.post(environment.api + '/searchrequest', body).subscribe((data: any) => alert(data.queries));
```

Comment:
Zou dit niet `data.vacancies` moeten zijn in plaats van `data.queries`?

Reply:
Voor nu laten we enkel de zoekopdracht zien puur om te bewijzen dat de connectie werkt. Dit wordt later omgebouwd naar een target solution met een lijst van vacatures